radis/misc/__init__.py

```diff
@@ -48,3 +48,9 @@
 from .progress_bar import ProgressBar
 from .signal import resample, resample_even
 from .utils import DatabankNotFound, NotInstalled, getProjectRoot
+
+# Checking `~radis.json` exist or not, if not then converting `~.radis` into `~/radis.json`
```

Comment:
What about moving this in [config.py/getConfig](https://github.com/radis/radis/blob/7a5b9b3179c6fc75eea626e98cf3e3e0810ff46c/radis/misc/config.py#L51) , if jsonfile is not found ?

Reply:
Yes, definitely a better approach!